src/components/ActionButton/ActionButton.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../sass/Variables.scss';
+
+$primary-color-lightest: #ffffff;
+$primary-color-lightest-hover: whitesmoke;
+$primary-color-light: #e5e5e5;
+$primary-color-dark: #4a4a4a;
+$accent-color: $calypso;
+
+button.CoveoActionButton.coveo-actionbutton {
+    height: 36px;
+    border: solid 2px $primary-color-light;
+    border-radius: 3px;
+    background-color: $primary-color-lightest;
+    color: $primary-color-dark;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    white-space: nowrap;
+}
+
+.CoveoActionButton {
+    &:not(.coveo-actionbutton-icon) {
+        padding: 0 16px;
+    }
+
+    &.coveo-actionbutton-icon {
+        width: 36px;
+    }
+
+    .coveo-actionbutton_icon {
```

Comment:
underscore is probably meant to be hyphen.

Reply:
No, `coveo-actionbutton-icon` and `coveo-actionbutton_icon` do not represent the same thing. `coveo-actionbutton-icon` means that the action button is rendering in icon-only mode. The styling is a bit different in this case to ensure the button is a square and its proportions do not change depending on the icon proportions.

On the other hand `coveo-actionbutton_icon` represents the element that contains the icon part of the button. That's why we also have `coveo-actionbutton_title` to represent the title part of the button.

Reply:
Is it common practice to make classes distinct by swapping - for _ ? It seems like it can lead to confusion.

Reply:
I get your concern. What would you think of `coveo-actionbutton-icononly`?
I would prefer having a single word to represent the "icon only" thing. But, at this moment, I don't know.

Reply:
Ah i see! I kind of like the idea of using the _ to further signify a "mode" or "type". So `coveo-actionbutton_icononly ` seems good too. My only concern is when we have two classes with names that only differ by the "-" or " _".

Reply:
Good concerns 👍
Naming is hard 😛 
I'll stop my Haiku here. 🎉